test/functional/blockchain.py

```diff
@@ -56,6 +56,22 @@ def _test_getchaintxstats(self):
         # we have to round because of binary math
         assert_equal(round(chaintxstats['txrate'] * 600, 10), Decimal(1))
 
+        b1 = self.nodes[0].getblock(self.nodes[0].getblockhash(1))
+        b200 = self.nodes[0].getblock(self.nodes[0].getblockhash(200))
+        ntimediff = b200['mediantime'] - b1['mediantime']
```

Comment:
nit: name the variable `time_diff`. (the `n` prefix in the C++ code is no longer the style we use, and shouldn't be carried over to the python code)